test/test_dataframe.py

```
@@ -1849,6 +1841,218 @@ def test_f64_np_array(self):
+                    pd.Timestamp(2023, 2, 3, 15, 45, 30)
+                ], dtype='datetime64[us]'),
+                'ts2': pd.Series([
+                    pd.Timestamp(2023, 2, 1, 10, 0, 0),
+                    None,
+                    pd.Timestamp(2023, 2, 3, 15, 45, 30)
+                ], dtype='datetime64[us]')
+            })
+
+            act = _dataframe(self.version, df, table_name='tbl1', at='ts2')
+
+            # format designated timestamp micros
```

Ruff (0.14.5)

Comment contains ambiguous ` ` (NO-BREAK SPACE). Did you mean ` ` (SPACE)?

(RUF003)